azure-ml-deployment/Python/config.txt

```diff
@@ -0,0 +1,15 @@
+workspace_name="DAI_DEMO2"
+resource_group_name="DAI_DEPLOY2"
+location="eastus"
+deployment_type="ACI"
+service_name="mydemoserviceaci"
+aks_cluster="myaks"
+model_name="MyDaiModel"
+description="Credit Card Demo"
+model_path="../mojo/pipeline.mojo"
+daimojo_path="../scoring-pipeline/env/dependencies/daimojo-2.0.1+master.478-cp36-cp36m-linux_x86_64.whl"
```

Comment:
is this example config? if yes, then should be in "example" folder, if it is config template, there should be placeholders instead of the hard-coded values.